jenkins-pipelines/manager/debian11-manager-install.jenkinsfile

```diff
@@ -9,6 +9,8 @@ managerPipeline(
     test_name: 'mgmt_cli_test.ManagerInstallationTests.test_manager_installed_and_functional',
     test_config: '''["test-cases/manager/manager-installation-set-distro.yaml", "configurations/manager/debian11.yaml"]''',
 
+    scylla_version: '2024.1',
```

Comment:
shouldn't be 2024.2?

Reply:
I put `2024.1` there by purpose for debian jobs to keep some coverage for `2024.1` too since it's still used in Production.